### PR TITLE
Update fossa config, add fossa make target

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:
Updates the fossa analysis configuration file, adds the make target and script to run fossa analysis.
Also updates the FOSSA link in the README.md to point to the new FOSSA project.

Presubmit and postsubmit created with this PR: https://github.com/kubevirt/project-infra/pull/1262

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .fossa.yml [hunk 1]
```diff
@@ -8,90 +8,87 @@ cli:
...
     options:
       modules-vendor: true
+      strategy: gomodules
   - name: virtctl
     type: go
     target: 'kubevirt.io/kubevirt/cmd/virtctl'
     path: cmd/virtctl
     options:
       modules-vendor: true
+      strategy: gomodules
```
Comment (rmohr) [inline .fossa.yml:66]: does this strategy limit the analysis to actually used dependencies?

Comment (dhiller) [reply .fossa.yml:66]: IIUC the [docs here](https://github.com/fossas/fossa-cli/blob/master/docs/integrations/golang.md#strategy-string), then yes, it does. Finally :)

**Pull request merged**